mobile/lib/services/rest_api.dart

```diff
@@ -165,36 +165,34 @@ class AirqoApiClient {
     return json.decode(response.body)['status'] as bool;
   }
 
-  Future<InsightData> fetchInsightsData(String siteId) async {
+  Future<List<Forecast>> fetchForecast(String siteId) async {
     try {
-      final now = DateTime.now();
-      final utcNow = now.toUtc();
-      final startDateTime = utcNow.getFirstDateOfCalendarMonth().toApiString();
-      final endDateTime = '${DateFormat('yyyy-MM-dd').format(
-        utcNow.getLastDateOfCalendarMonth(),
-      )}T23:59:59Z';
-
-      final queryParams = <String, dynamic>{
-        'siteId': siteId,
-        'utcOffset': now.getUtcOffset(),
-        'startDateTime': startDateTime,
-        'endDateTime': endDateTime,
-      };
+      final body =
+          await _performGetRequest({}, "${AirQoUrls.forecast}/$siteId");
 
-      final body = await _performGetRequest(
-        queryParams,
-        AirQoUrls.insights,
-      );
+      final forecast = <Forecast>[];
+
+      for (final prediction in body['predictions']) {
+        try {
+          forecast.add(
+            Forecast.fromJson({
+              'pm2_5': prediction['prediction_time'],
+              'time': prediction['prediction_value'],
+              'siteId': siteId,
+            }),
```

Comment:
@Mnoble-19 expecting the daily forecast endpoint to return to return data with at least `prediction_time` and `prediction_value`